src/components/tasks/card/index.tsx

```diff
@@ -529,37 +487,15 @@ const Card: FC<CardProps> = ({
                     </span>
                     {isEditable ? (
                         isUserAuthorized && (
-                            <div className={classNames.suggestionDiv}>
-                                <input
-                                    data-testid="assignee-input"
-                                    ref={inputRef}
-                                    value={assigneeName}
-                                    className={classNames.cardStrongFont}
-                                    onKeyDown={(e) => {
-                                        handleChange(e, 'assignee');
-                                    }}
-                                    onChange={(e) => {
-                                        handleAssignment(e);
-                                        debounce(
-                                            fetchUsers,
-                                            400
-                                        )(e.target.value);
-                                    }}
-                                    role="button"
-                                    tabIndex={0}
-                                />
-
-                                {isLoadingSuggestions ? (
-                                    <Loader />
-                                ) : (
-                                    showSuggestion && (
-                                        <SuggestionBox
-                                            suggestions={suggestions}
-                                            onSelectAssignee={handleClick}
-                                        />
-                                    )
-                                )}
-                            </div>
+                            <Suggestions
+                                assigneeName={assigneeName}
+                                searchTerm={debounceSearchTerm}
+                                showSuggestion={showSuggestion}
+                                handleAssignment={handleAssignment}
+                                handleClick={handleClick}
+                                handleChange={handleChange}
+                                ref={inputRef}
```

Comment:
Are these handlers  & state used anywhere within card component? If not, please move it to the Suggestions component to avoid adding unwanted handlers and state. 

For e.g. showSuggestion, I don't see the use of this state anywhere within this card component. 
Did you check if this change creates any unused state and set state and aother handlers if you don't pass them as props. If yes, they aren't required here. Please ensure this and only then keep them here.

Reply:
I don't see assigneeName, setAssigneeName being used here. Even the click handlers, debounced search term - please move them to the specific component.

Reply:
@bharati-21 showSuggestion is being passed as prop to Suggestions, but it is being toggled in handleAssignment and handleClick thats why they reside in the index.tsx instead of in Suggestions. 
Also setAssigneeName is also being used in handleAssignment in index.tsx